bin/express

```diff
@@ -398,18 +448,20 @@ function main() {
   var appName = createAppName(path.resolve(destinationPath)) || 'hello-world'
 
   // View engine
-  if (program.view === undefined) {
+  if (program.view) {
```

Comment:
I'm not sure if it's this line, but something around here broke the preference for `--view=<engine>` over the old `--<engine>` switches. Currently, if you call `express --ejs --view=pug`, we use the newer variant (pug is used). This PR reveres this for some reason and now causes the same command to generate with ejs.